src/screens/profile/styles.ts

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components/native';
+
+export const HeadingContainer = styled.Text`
+  margin-top: 86px;
+  margin-bottom: 16px;
+  word-break: break-all;
+  width: 100%;
+`;
+export const ButtonContainer = styled.TouchableOpacity`
+  margin-top: 30%;
+  margin-left: 30px;
+`;
+export const ButtonPasswordContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
+export const ButtonEmailContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
+export const ButtonPhoneContainer = styled.TouchableOpacity`
+  margin-top: 4%;
+  margin-left: 0px;
+`;
```

Comment:
Since these styles are exactly the same, we can just use one component for all three buttons.